mproton.go

```diff
@@ -0,0 +1,101 @@
+package mproton
+
+/*
+#cgo darwin CFLAGS: -x objective-c
+#cgo darwin CFLAGS: -Wobjc-property-no-attribute
+#cgo darwin LDFLAGS: -framework Cocoa
+#cgo darwin LDFLAGS: -framework Foundation
+#cgo darwin LDFLAGS: -framework WebKit
+
+//sudo apt-get -y install libgtk-3-dev libwebkit2gtk-4.0-dev libappindicator3-dev
+
+#cgo linux openbsd freebsd CXXFLAGS: -DWEBVIEW_GTK -std=c++11
+#cgo linux openbsd freebsd pkg-config: gtk+-3.0 webkit2gtk-4.0 appindicator3-0.1
+
+
+// sudo apt-get install -y electric-fence
+// #cgo linux LDFLAGS: -lefence
+
+#include "mproton.h"
+
+
+#include <stdio.h>
+#include <stdlib.h>
+*/
+import "C"
+import "fmt"
+
+var callbackMap = make(map[string]func(v string) (string, error))
```

Comment:
Probably will need a mutex around it. If it becomes a bottleneck, there are options to remove get rid of it.

Reply:
Added.